src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -223,35 +203,24 @@ void RotatedSPOs::readVariationalParameters(hdf_archive& hin)
   if (grp_global_exists)
   {
     hin.push("rotation_global", false);
-    std::string rot_global_name = std::string("rotation_global_") + SPOSet::getName();
+    const std::string rot_global_name = std::string("rotation_global_") + SPOSet::getName();
 
     std::vector<int> sizes(1);
-    if (!hin.getShape<RealType>(rot_global_name, sizes))
+    if (!hin.getShape<ValueType>(rot_global_name, sizes))
```

Comment:
When using ValueType as ComplexType, the last dimension of complex {2} gets excluded when inquering the shape.